lab_5_scrapper/scrapper.py

```diff
@@ -100,6 +211,12 @@ def make_request(url: str, config: Config) -> requests.models.Response:
     Returns:
         requests.models.Response: A response from a request
     """
+    return requests.get(
```

Comment:
remember that you need to make requests and make sleeps for some period (timeout). otherwise it is obvious you are not a browser user

Reply:
I pass it to the .get function. Or should I call something like time.sleep()?

Reply:
Okay, I revised lecture and seminar and added time.sleep